src/pkgdev/scripts/pkgdev_mask.py

```diff
@@ -99,6 +114,18 @@ def __str__(self):
         lines.extend(map(str, self.atoms))
         return '\n'.join(lines)
 
+    def send_email(self):
+        try:
+            subprocess.run(args=[
+                'xdg-email', '--utf8',
+                '--cc', 'gentoo-dev@lists.gentoo.org',
+                '--subject', 'Last rites: ' + ', '.join(map(str, self.atoms)),
```

Comment:
I'd probably move the python code out into a separate variable for the atom list string and then use an f-string to combine it here.